runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsBrotherSlainOverlay.java

```diff
@@ -62,11 +65,16 @@ public Dimension render(Graphics2D graphics)
 
 		// Hide original overlay
 		Widget barrowsBrothers = client.getWidget(WidgetInfo.BARROWS_BROTHERS);
-		if (barrowsBrothers != null)
+		if (barrowsBrothers != null || !config.showBrotherOverlay())
```

Comment:
this isn't right

Reply:
What’s wrong? It worked fine when I tested it.

Reply:
What’s wrong? It worked fine when I tested it, unless you want to keep the original overlay.